assets/js/lists/list-creation.js

```diff
@@ -21,16 +21,22 @@ storefrontApp.controller('recentlyCreateNewListDialogController', ['$rootScope',
         $scope.selectedList.items = items;
     };
 
+    $scope.isSelected = function () {
```

Comment:
incorrect name of the method.
It should be noneSelected in that implemenation.
But better to make it `isAnySelected`  and use !isAnySelected in ng-disabled

Reply:
fixed